src/AwsClient.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace SublimeSkinz\SublimeVault;
+
+use Aws\S3\S3Client;
+use Aws\S3\Exception\S3Exception;
+use Analog\Logger;
+use Analog\Handler\File;
+
+class AwsClient
+{
+    protected $logger;
+
+    public function __construct()
+    {
+        $logger = new Logger();
+        $logger->handler(File::init(__DIR__ . '/../logs/errors.log'));
```

Comment:
please don't hard code the path for the log, I think the best default is `/dev/stdout` / `/dev/stderr` but if we can expose a stream log is better I don't know if it's possible in php